.github/workflows/build.yml

```diff
@@ -50,6 +50,23 @@ jobs:
       - run: pnpm install
       - run: pnpm audit
 
+  assign-author:
```

Comment:
It looks like this step is being run whenever we merge a PR to `main`. https://github.com/iTwin/iTwinUI/actions/runs/10910728547/job/30281834580

We can add some `if` check here, or even move this step to a different job or workflow altogether, since it's unrelated to all the other `build` steps.

Reply:
I will open another PR addressing this issue.